### PR TITLE
Submitting in class

### PR DESCRIPTION
**Pull request by dhasler**

### CODE REVIEW

File: css/icOne.css [hunk 1]
```diff
@@ -1,3 +1,101 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto';
```
Comment (matty-digital) [inline css/icOne.css]: no imports in vanilla CSS, I'm afraid

File: css/icOne.css [hunk 2]
```diff
@@ -1,3 +1,101 @@
...
+
 body, html {
-  font-size: 12px;
+  font-size: 62.5%;
+  font-family: 'Roboto', sans-serif;
+}
+
+h1{
```
Comment (matty-digital) [inline css/icOne.css]: space between selector and curly brace

File: css/icOne.css [hunk 3]
```diff
@@ -1,3 +1,101 @@
...
+
+nav ul li a:before{
+  content:"\0227C";
+  display: block;
+  left: 0;
+  opacity: 0;
+  
+  position: absolute;
+  top: 0;
+  transition: all 0.2s;
```
Comment (matty-digital) [inline css/icOne.css]: 🎩

File: index.html [hunk 4]
```diff
@@ -2,10 +2,76 @@
...
+      <div class="center">
+        <section id="directDrive">
+          <h1>KOENIGSEGG DIRECT DRIVE</h1>
+          <p>Traditional hybrids represent a compromise when it comes to weight, complexity, cost, packaging and efficiency.  
+          The Regera is not what we at Koenigsegg would call a traditional hybrid as it does not have the traditional shortcomings of a hybrid. Instead, the Regera is a new breed of hybrid. It is a new breed of Koenigsegg and a new breed of car, for that matter.
 
+          
+            Traditional ‘parallel’ hybrids are compromised and heavy as they have two independent propulsion systems moving the car. While ‘series’ hybrids are less compromised when it comes to weight, complexity and costs, they are compromised when it comes to efficiency as there is too much energy conversion taking place.
+            <img src="http://1hosj01irnixn8onr1zmv5s1.wpengine.netdna-cdn.com/wp-content/uploads/2015/03/Koenigsegg_Regera_Crane_H8A3443_LR.jpg" alt="Koenigsegg Regera" title="Koenigsegg Regera" />           
+
```
Comment (matty-digital) [inline index.html]: careful with the formatting

File: index.html [hunk 5]
```diff
@@ -2,10 +2,76 @@
...
+            </p>
+            <p>
+              <label for="checkA">Select A</label>
+              <input id="checkA" type="checkbox" name="checkboxes" value="A" />
+              <label for="checkB">Select B</label>
+              <input id="checkB" type="checkbox" name="checkboxes" value="B" />
+              <label for="checkC">Select C</label>
+              <input id="checkC" type="checkbox" name="checkboxes" value="C" />
+            </p>
+            <textarea id="textTextarea" placeholder="">
```
Comment (matty-digital) [inline index.html]: does `textarea` require a closing tag?

**Comment by matty-digital** [comment]
👍

File: index.html (hunk 5 above)
Comment (dhasler) [inline index.html]: the W3C has it in their stuff for textarea.  
http://www.w3schools.com/tags/tag_textarea.asp
If it is not required or is better, I will definitely remember that!

Comment (matty-digital) [inline index.html]: Right on. It didn't look wrong I was mostly curious as I don't think I've used one for a while. Keep it this way I appreciate the research you did

**Pull request closed without merging**